Dockerfile

```diff
@@ -1,27 +1,28 @@
 # Dockerfile
 # ---- Base Node ----
 FROM node:20.11.0 AS base
-ENV NEXT_TELEMETRY_DISABLED 1
-WORKDIR /app
-COPY apps/web-staking/package.json ./apps/web-staking/
-COPY apps/web-staking/package-lock.json ./apps/web-staking/
-RUN npm install --loglevel verbose -dd --prefix ./apps/web-staking
+
+# TODO => Remove this line for production deployment
+ENV NEXT_PUBLIC_APP_ENV=development
+
+# Install PNPM
+RUN npm install -g pnpm
+
+# Install dependencies with PNPM
+RUN pnpm install --loglevel verbose --filter ./apps/web-staking
```

Comment:
I think we should be isntalling the whole monorepo, in the future we will want to use the ui library that could import packages from the ui library. Is there any downside to just doing `pnpm install` ?

Reply:
Done!